v2/internal/controllers/samples_test.go

```diff
@@ -40,6 +40,11 @@ var randomNameExclusions = []string{
 
 func Test_Samples_CreationAndDeletion(t *testing.T) {
 	t.Parallel()
+
+	if *isLive {
```

Comment:
Don't you need a **nil** check before dereferencing `isLive` for safety?

Reply:
default for `isLive` would always be `false`. So it never ends up being nil